tethysext/atcore/public/gizmos/slide_sheet/slide_sheet.js

```diff
@@ -26,11 +26,15 @@ var SLIDE_SHEET = (function() {
     *                    PRIVATE FUNCTION IMPLEMENTATIONS
     *************************************************************************/
     open = function(id) {
-        $('#' + id + '.slide-sheet').addClass('show');
+        if (id.length) {
```

Comment:
For some reason, tethys sometimes tries to load the slide-sheet with no id given (empty string). This will cause the rest of the javascript to stop running. I put the if statement there to insure that the id must be specified to show the slidesheet.